bitnami/kubeapps/README.md

```diff
@@ -51,7 +52,417 @@ Once you have installed Kubeapps follow the [Getting Started Guide](https://gith
+| `pinnipedProxy.image.tag`                             | Pinniped Proxy image tag (immutable tags are recommended)                | `2.3.2-debian-10-r0`              |
+| `pinnipedProxy.image.pullPolicy`                      | Pinniped Proxy image pull policy                                         | `IfNotPresent`                    |
+| `pinnipedProxy.image.pullSecrets`                     | Pinniped Proxy image pull secrets                                        | `[]`                              |
+| `pinnipedProxy.defaultPinnipedNamespace`              | Specify the (default) namespace in which pinniped concierge is installed | `pinniped-concierge`              |
+| `pinnipedProxy.defaultAuthenticatorType`              | Specify the (default) authenticator type                                 | `JWTAuthenticator`                |
+| `pinnipedProxy.defaultAuthenticatorName`              | Specify the (default) authenticator name                                 | `jwt-authenticator`               |
+| `pinnipedProxy.defaultPinnipedAPISuffix`              | Specify the (default) API suffix                                         | `pinniped.dev`                    |
+| `pinnipedProxy.containerPort`                         | Kubeops HTTP container port                                              | `3333`                            |
+| `pinnipedProxy.containerSecurityContext.enabled`      | Enabled Pinniped Proxy containers' Security Context                      | `true`                            |
+| `pinnipedProxy.containerSecurityContext.runAsUser`    | Set Pinniped Proxy container's Security Context runAsUser                | `1001`                            |
+| `pinnipedProxy.containerSecurityContext.runAsNonRoot` | Set Pinniped Proxy container's Security Context runAsNonRoot             | `true`                            |
+| `pinnipedProxy.resources.limits.cpu`                  | The CPU limits for the Pinniped Proxy container                          | `250m`                            |
+| `pinnipedProxy.resources.limits.memory`               | The memory limits for the Pinniped Proxy container                       | `128Mi`                           |
+| `pinnipedProxy.resources.requests.cpu`                | The requested CPU for the Pinniped Proxy container                       | `25m`                             |
+| `pinnipedProxy.resources.requests.memory`             | The requested memory for the Pinniped Proxy container                    | `32Mi`                            |
+
+
+### Other Parameters
+
+| Name                    | Description                                        | Value                   |
+| ----------------------- | -------------------------------------------------- | ----------------------- |
+| `rbac.create`           | Specifies whether RBAC resources should be created | `true`                  |
```

Comment:
Woah - is that from before all clusters had RBAC enabled? Looks like it was added [here](https://github.com/kubeapps/kubeapps/pull/483). Happy to delete that option?

Reply:
According to the section below, the chart should work on K8s 1.15+:
```
## Prerequisites

- Kubernetes 1.15+ (tested with Azure Kubernetes Service, Google Kubernetes Engine, minikube and Docker for Desktop Kubernetes)
```
@absoludity can you confirm that RBAC is enabled on every cluster since that version (including custom on-premises clusters)?

Reply:
Erm, no? How could I possibly do that? If this is a standard option in all bitnami charts ([looks like it might be](https://github.com/bitnami/charts/search?q=rbac.create&unscoped_q=rbac.create)) - great, I have no problem with it. I had just looked up the commit and thought it was added specifically to Kubeapps for tiller support.

Reply:
Let's keep it then. I don't think it makes no harm